content/terraform/v1.13.x (beta)/docs/language/tests/index.mdx

```diff
@@ -214,7 +241,7 @@ run "overrides_root_level_value" {
 }
 ```
 
-We've added a second `run` block that specifies the `bucket_prefix` variable value as `other`, overriding the value `test` that is provided by the test file and used during the first `run` block.
+Instead of needing external input to provide a value for our tests, we can define our variables directly within the test file. We've also added a second `run` block that specifies the `bucket_prefix` variable value as `other`, overriding the value `test` that is provided by the test file and used during the first `run` block.
```

Comment:
```suggestion
Instead of needing external input to provide a value for our tests, you can define our variables directly within the test file. This example also adds a second `run` block that specifies the `bucket_prefix` variable value as `other`, overriding the value `test` that is defined in the test file and used during the first `run` block.
```